drivers/hmis/config/initializers/strict_attributes.rb

```diff
@@ -0,0 +1,37 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+# discussed with Gig and agreed that it would be more clear to put this in the models themselves (aka in a concern)
+# rather than here in a hook. However, I'm leaving this in the PR for discussion since I need to better understand rails model loading.
+# In this strategy, as long as we call Rails.application.eager_load!, all the models' tables are loaded.
+
+# # if changing this code and then testing it using rspec, need to DISABLE_SPRING=1
+# Rails.application.config.after_initialize do
+#   # without eager_load, Hmis::Hud::Base.descendants == [].
+#   Rails.application.eager_load!
```

Comment:
Approach 3: In an `after_initialize` hook.
Cons of this approach: 
- It's less clear because it defines model code somewhere far-ish away from the models themselves.
- I don't really know about the ramifications of calling `Rails.application.eager_load!` here. Will that have bad performance effects on application load?

Pro of this approach:
- Using eager loading, it's the only approach I've managed to find so far that actually defines the attributes on all of the models.